src/openms/include/OpenMS/FILTERING/TRANSFORMERS/LinearResamplerAlign.h

```diff
@@ -164,13 +213,59 @@ namespace OpenMS
         @param resample_it Iterator pointing to end of the output spectrum range (m/z need to be populated, intensities should be zero)
 
     */
-    template <typename PeakTypeIterator, typename ConstPeakTypeIterator>
-    void raster(ConstPeakTypeIterator raw_it, ConstPeakTypeIterator raw_end, PeakTypeIterator resample_it, PeakTypeIterator resample_end)
+    void raster(MSChromatogram::ConstIterator raw_it,
```

Comment:
why not just leave the template here? seems like a copy-paste issue here that is perfectly suited for a template?